src/content/en/updates/2019/07/chrome-75-media-updates.md

```diff
@@ -0,0 +1,175 @@
+`navigator.mediaCapabilities.decodingInfo()` returns a promise that resolves
+asynchronously with an object containing three booleans: `supported`, `smooth`,
+and `powerEfficient`. However when a`keySystemConfiguration` key is set and
+`supported` is `true`, yet another MediaKeySystemAccess object named
+`keySystemAccess` is returned as well. It can be used to request some media keys
+and setup encrypted media playback. Here’s an example.
+
+    // Like rMSKA(), orderedMediaConfigs is ordered from most to least wanted.
+    const capabilitiesPromises = orderedMediaConfigs.map(mediaConfig =>
+      navigator.mediaCapabilities.decodingInfo(mediaConfig)
+    );
+
+    // Assume this app wants a supported and smooth media playback.
+    let bestConfig = null;
+    for await (const result of capabilitiesPromises) {
+      if (!result.supported || !result.smooth) {
+        continue;
+      }
+      bestConfig = result;
+      break;
+    }
+
+    if (bestConfig) {
+      const mediaKeys = await bestConfig.keySystemAccess.createMediaKeys();
+      // TODO: rest of EME path as-is
+    } else {
+      // Argh! No smooth configs found.
+      // TODO: Maybe choose the lowest resolution and framerate available.
+    }
+
+Note that the Decoding Info API for encrypted media requires HTTPS.
+
+Moreover, be aware that it may trigger a user prompt on Android and Chrome OS in
+the same way as rMKSA. It won’t show more prompts than rMKSA though, in spite of
+requiring more calls to setup encrypted media playback.
+
+
+<div class="attempt-right">
+  <figure>
+    <img src="/web/updates/images/2019/07/protected-content-prompt.jpg"
+        alt="Protected content prompt">
+    <figcaption>
+      <b>Figure 1.</b>
+      Protected content prompt
+    </figcaption>
+  </figure>
+</div>
+<div class="clearfix">
+</div>
+
+Dogfood: To get feedback from web developers, this feature is available as an
+[Origin Trial] in Chrome 75. You will need to [request a token], so that the
+feature is automatically enabled for your origin for a limited period
+of time. This will eliminate the need to enable the "Web Platform Features"
+flag.
+
+[Intent to Experiment](https://groups.google.com/a/chromium.org/d/topic/blink-dev/eA9uG98td5U/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5765900795904000) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=907909)
+
+## HTMLVideoElement.playsInline {: #playsinline}
+
+Chrome now supports the `playsInline` boolean attribute. If present, it hints to
+the browser that the video ought to be displayed "inline" in the document by
+default, constrained to the element's playback area.
+
+[Similarly] to Safari where video elements on iPhone don’t automatically enter
+fullscreen mode when playback begins, this hint allows some embedders to have an
+auto-fullscreen video playback experience. Web developers can use it to opt-out
+of this experience if needed.
+
+    <video playsinline></video>
```

Comment:
Shouldn't this be inside a ```` ```html ... ``` ```` block?

Reply:
It doesn't need to. See https://pr-7920-dot-web-central.appspot.com/web/updates/2019/07/chrome-75-media-updates